src/util/safelywritablefile.cpp

```diff
@@ -46,40 +56,10 @@ SafelyWritableFile::SafelyWritableFile(QString origFileName, bool useTemporaryFi
         // Abort constructor
         return;
     }
-    if (useTemporaryFile) {
-        QString tempFileName = origFileName + kSafelyWritableTempFileSuffix;
-        QFile origFile(origFileName);
-        if (!origFile.copy(tempFileName)) {
-            kLogger.warning()
-                    << origFile.errorString()
-                    << "- Failed to clone original into temporary file before writing:"
-                    << origFileName
-                    << "->"
-                    << tempFileName;
-            // Abort constructor
-            return;
-        }
-        QFile tempFile(tempFileName);
-        DEBUG_ASSERT(tempFile.exists());
-        // Both file sizes are expected to be equal after successfully
-        // copying the file contents.
-        VERIFY_OR_DEBUG_ASSERT(origFile.size() == tempFile.size()) {
-            kLogger.warning() << "Failed to verify size after cloning original "
-                                 "into temporary file before writing:"
-                              << origFile.size() << "<>" << tempFile.size();
-            // Cleanup
-            if (tempFile.exists() && !tempFile.remove()) {
-                kLogger.warning()
-                        << tempFile.errorString()
-                        << "- Failed to remove temporary file:"
-                        << tempFileName;
-            }
-            // Abort constructor
-            return;
-        }
-        // Successfully cloned original into temporary file for writing - finish initialization
-        m_origFileName = std::move(origFileName);
-        m_tempFileName = std::move(tempFileName);
+    m_origFileName = std::move(origFileName);
```

Comment:
This is set in the else branch as well. After std::move the origFileName is a default constructed QString()

Reply:
Oh yes, sorry I have missed that. No need to do it in else branch. Thank you for noticing me 👍